octotiger/node_server.hpp

```diff
@@ -128,9 +128,9 @@ class OCTOTIGER_EXPORT node_server: public hpx::components::managed_component_ba
 	}
 	real get_rotation_count() const;
 	node_server& operator=(node_server&&) = default;
-	static std::uint16_t cumulative_nodes_count(bool);
-	static std::uint16_t cumulative_leafs_count(bool);
-	static std::uint16_t cumulative_amrs_count(bool);
+	static int cumulative_nodes_count(bool);
+	static int cumulative_leafs_count(bool);
+	static int cumulative_amrs_count(bool);
```

Comment:
Using `int` will still restrict things to 32 bits. I'd suggest you use `std::int64_t` instead (like all other performance counters).